CONTRIBUTING.md

```diff
@@ -12,19 +12,19 @@ New features are welcome! Either as requests or proposals.
 
 1. Please create an issue first, so we know what to expect from you.
 2. Create a fork on your GitHub account.
-4. Make the changes or additions in your fork.
-5. Please create or edit unit tests or integration tests.
-6. Run the tests and verify they pass.
-7. Update documentation.
-8. [Create](https://github.com/stryker-mutator/stryker-dashboard/compare) a Pull Request.
+3. Make the changes or additions in your fork.
```

Comment:
If you change this list to:

```suggestion
1.
1.
1.
1. Make the changes or additions in your fork.
```

1.
1.
1.
1.

Github computes the list numbers automatically.

Reply:
Linter is not happy with that unfortunately

Reply:
Luckily we have a `lint:fix` command 😎

Reply:
> Linter is not happy with that unfortunately

Then the linter is wrong

Reply:
I don't mind it 😅